docs_user/modules/proc_migrating-ovn-data.adoc

```diff
@@ -102,31 +98,31 @@ spec:
 EOF
 ----
 
-. Wait for the pod to come up
+. Wait for the pod to be ready:
 +
 ----
-oc wait --for=condition=Ready pod/ovn-copy-data --timeout=30s
+$ oc wait --for=condition=Ready pod/ovn-copy-data --timeout=30s
 ----
 
-. Backup OVN databases on an environment without TLS everywhere.
+. Back up your OVN databases:
+* If you did not enable TLS everywhere, run the following command:
 +
 ----
-oc exec ovn-copy-data -- bash -c "ovsdb-client backup tcp:$SOURCE_OVSDB_IP:6641 > /backup/ovs-nb.db"
-oc exec ovn-copy-data -- bash -c "ovsdb-client backup tcp:$SOURCE_OVSDB_IP:6642 > /backup/ovs-sb.db"
+$ oc exec ovn-copy-data -- bash -c "ovsdb-client backup tcp:$SOURCE_OVSDB_IP:6641 > /backup/ovs-nb.db"
+$ oc exec ovn-copy-data -- bash -c "ovsdb-client backup tcp:$SOURCE_OVSDB_IP:6642 > /backup/ovs-sb.db"
 ----
-
-. Alternatively, backup OVN databases on a TLS everywhere environment.
+* If you enabled TLS everywhere, run the following command:
 +
 ----
-oc exec ovn-copy-data -- bash -c "ovsdb-client backup --ca-cert=/etc/pki/tls/misc/ca.crt --private-key=/etc/pki/tls/misc/tls.key --certificate=/etc/pki/tls/misc/tls.crt ssl:$SOURCE_OVSDB_IP:6641 > /backup/ovs-nb.db"
-oc exec ovn-copy-data -- bash -c "ovsdb-client backup --ca-cert=/etc/pki/tls/misc/ca.crt --private-key=/etc/pki/tls/misc/tls.key --certificate=/etc/pki/tls/misc/tls.crt ssl:$SOURCE_OVSDB_IP:6642 > /backup/ovs-sb.db"
+$ oc exec ovn-copy-data -- bash -c "ovsdb-client backup --ca-cert=/etc/pki/tls/misc/ca.crt --private-key=/etc/pki/tls/misc/tls.key --certificate=/etc/pki/tls/misc/tls.crt ssl:$SOURCE_OVSDB_IP:6641 > /backup/ovs-nb.db"
+$ oc exec ovn-copy-data -- bash -c "ovsdb-client backup --ca-cert=/etc/pki/tls/misc/ca.crt --private-key=/etc/pki/tls/misc/tls.key --certificate=/etc/pki/tls/misc/tls.crt ssl:$SOURCE_OVSDB_IP:6642 > /backup/ovs-sb.db"
 ----
 
-. Start control plane OVN database services prior to import, while keeping `northd` and `ovn-controller` disabled.
+. Start the control plane OVN database services prior to import, with `northd` and `ovn-controller` disabled:
 +
 [source,yaml]
 ----
-oc patch openstackcontrolplane openstack --type=merge --patch '
+$ oc patch openstackcontrolplane openstack-galera-network-isolation --type=merge --patch '
```

Comment:
Step 5: Should the number of replicas be 1? 
https://github.com/openstack-k8s-operators/architecture/pull/305/files

Reply:
I think it should be set like that already, this step is just to update it from "enabled: false" to "enabled:true"